packages/app/src/contacts/components/contact-import/contact-import-modal.component.tsx

```diff
@@ -82,18 +82,19 @@ export enum ModalType {
   Select,
 }
 
-interface Props {
-  onActionButtonClick: (contacts: NewContact[]) => void
+interface Props extends ComponentProps<typeof ModalDialog> {
+  handleActionButtonClick: (contacts: NewContact[]) => void
   contacts: NewContact[]
   modalType: ModalType
   successfulItemsCount?: number
 }
 
 const ContactImportModal: FunctionComponent<Props> = ({
-  onActionButtonClick,
+  handleActionButtonClick,
```

Comment:
I know, but in this case, `onActionButtonClick` is a reserved prop name for ModalDialog component. I need to use it a little bit differently, this is why I changed name. So it can't be `on` and it shouldn't be `handle`, do you have any other idea how to name it properly? 😄